core/services/accesskey/accesskey.proto

```diff
@@ -0,0 +1,100 @@
+syntax = "proto3";
+package erda.core.services.accesskey;
+option go_package = "github.com/erda-project/erda-proto-go/core/services/accesskey/pb";
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+service AccessKeyService {
+    rpc ListAccessKeys (ListAccessKeysRequest) returns (ListAccessKeysResponse) {
+        option (google.api.http) = {
+            get: "/api/credential/access-keys",
+        };
+    }
+
+    rpc GetAccessKey (GetAccessKeysRequest) returns (GetAccessKeysResponse) {
+        option (google.api.http) = {
+            get: "/api/credential/access-keys/{id}",
+        };
+    }
+
+    rpc CreateAccessKeys (CreateAccessKeysRequest) returns (CreateAccessKeysResponse) {
+        option (google.api.http) = {
+            post: "/api/credential/access-keys",
+        };
+    }
+
+    rpc UpdateAccessKeys (UpdateAccessKeysRequest) returns (UpdateAccessKeysResponse) {
+        option (google.api.http) = {
+            put: "/api/credential/access-keys/{id}",
+        };
+    }
+
+    rpc DeleteAccessKeys (DeleteAccessKeysRequest) returns (DeleteAccessKeysResponse) {
+        option (google.api.http) = {
+            delete: "/api/credential/access-keys/{id}",
+        };
+    }
+}
+
+message ListAccessKeysRequest {
+    optional GenericEnum.Status status = 1;
+    optional GenericEnum.SubjectType subjectType = 2;
+    optional string subject = 3;
```

Comment:
What does subject mean ?

Reply:
subject means the authenticate subject which could be a username, IAM-entity-id, organization-id, etc.

Reply:
It is recommended to use category or kind instead of subject